src/Agent.Listener/Configuration/FeatureFlagProvider.cs

```diff
@@ -49,7 +49,7 @@ public async Task<FeatureFlag> GetFeatureFlagAsync(IHostContext context, string
             var client = vssConnection.GetClient<FeatureAvailabilityHttpClient>();
             try
             {
-                return await client.GetFeatureFlagByNameAsync(featureFlagName);
+                return await client.GetFeatureFlagByNameAsync(featureFlagName, checkFeatureExists: false);
```

Comment:
*For future PR / not blocking* : would be good to add a comment why this is needed.  With this set, exception handler not needed?

Reply:
@merlynomsft  Yes, I will refactor this in next PR